test/interval_tests/construction.jl

```diff
@@ -1,18 +1,17 @@
-# This file is part of the IntervalArithmetic.jl package; MIT licensed
-
-using IntervalArithmetic
 using Test
+using IntervalArithmetic
+import IntervalArithmetic: unsafe_interval
```

Comment:
It would be nice to have test verifying that the now disallowed construction with `Interval` throw an error.

Reply:
Good idea. Done.